docker/run_activationtck.sh

```diff
@@ -59,10 +59,25 @@ if [ -z "$ANGUS_BUNDLE_URL" ];then
 fi
 wget $WGET_PROPS $ANGUS_BUNDLE_URL -O ${WORKSPACE}/angus-activation.jar
 
+if [ -z "$GF_BUNDLE_URL" ]; then
+  echo "Using default url for GF bundle: $DEFAULT_GF_BUNDLE_URL"
+  export GF_BUNDLE_URL=$DEFAULT_GF_BUNDLE_URL
+fi
+
+wget $WGET_PROPS $GF_BUNDLE_URL -O latest-glassfish.zip
+unzip -q -o latest-glassfish.zip
+
+TOP_GLASSFISH_DIR="glassfish6"
+chmod -R 777 ${TOP_GLASSFISH_DIR}
 
 sed -i "s#^TS_HOME=.*#TS_HOME=$TS_HOME#g" ${TS_HOME}/lib/ts.jte
 sed -i "s#^JAVA_HOME=.*#JAVA_HOME=$JAVA_HOME#g" ${TS_HOME}/lib/ts.jte
-sed -i "s#^JARPATH=.*#JARPATH=$WORKSPACE#g" ${TS_HOME}/lib/ts.jte
+
+if [[ "$RUNTIME" == "Glassfish" ]]; then
+  sed -i "s#^JARPATH=.*#JARPATH=$WORKSPACE/$TOP_GLASSFISH_DIR/glassfish/modules#g" ${TS_HOME}/lib/ts.jte
+else 
+  sed -i "s#^JARPATH=.*#JARPATH=$WORKSPACE#g" ${TS_HOME}/lib/ts.jte
+fi
```

Comment:
note there is also https://github.com/eclipse-ee4j/jaf-tck/blob/master/lib/ts.pluggability.jte for pluggability tests

Reply:
thanks @lukasj for the review, will add changes to ts.pluggability.jte, since plugability PR has now been merged.